auth/token.go

```diff
@@ -111,7 +111,7 @@ func GRPCGetIdentityFromIDToken(ctx context.Context, clientID string, provider *
 	}
 
 	meta := metautils.ExtractIncoming(ctx)
-	userInfoDecoded := DecodeFromBase64(meta.Get(UserInfoMDKey))
+	userInfoDecoded, _ := DecodeFromBase64(meta.Get(UserInfoMDKey))
```

Comment:
not sure if we really should drop the error. maybe we should move the log statement here?

Reply:
Sure let me take this in a followup PR.